bundle/artifacts/whl/autodetect.go

```diff
@@ -27,9 +27,9 @@ func (m *detectPkg) Name() string {
 }
 
 func (m *detectPkg) Apply(ctx context.Context, b *bundle.Bundle) error {
-	wheelTasks := libraries.FindAllWheelTasks(b)
+	wheelTasks := libraries.FindAllLocalWheelTasks(b)
```

Comment:
To clarify. With the `bundle run` extensions that @kartikgupta-db is working on, we might just have local execution as well, so `local wheel task` can end up being ambiguous or for actual local execution.

```suggestion
	wheelTasks := libraries.FindAllWheelTasksWithLocalLibraries(b)
```